pyqt-apps/siriushla/widgets/log_label.py

```diff
@@ -1,10 +1,15 @@
-import datetime as _datetime
+"""Log label."""
+
 from qtpy.QtWidgets import QListWidget, QListWidgetItem
 from qtpy.QtCore import Property, Q_ENUMS
 from qtpy.QtGui import QColor
+
+from pydm.data_plugins import plugin_for_address
 from pydm.widgets.base import PyDMWidget, TextFormatter
 from pydm.widgets.display_format import DisplayFormat, parse_value_for_display
 
+from siriuspy.clientarch import Time as _Time
```

Comment:
@anacso17 , since this `Time` class is being useful outside `clientarch`, should we not move it to a upper module in `siriuspy`, like `callbacks.py` and others? `Time` seems generic as it is and may be of general use. what do you think?